src/Routes/Lists.js

```diff
@@ -221,30 +210,13 @@ class ListsRoute extends BaseRoute {
                         status: 404,
                         message: 'The page you were looking for could not be found.'
                     });
-                    let changes = {};
-                    const validate = FormValidator.validateFeature(req.body);
-                    if (validate.length) return res.render('lists/features/edit', {
-                        title: `Edit Feature '${features[0].name}'`,
+                    const handle = await this.featureContoller.handle(req.body, true, features[0]);
+                    if (Array.isArray(handle)) return res.render('lists/features/edit', {
+                        title: 'Edit Feature "' + features[0].name + '"',
                         data: features[0],
-                        errors: validate
-                    });
-                    const columns = Object.keys(await this.db('features').columnInfo());
-                    for (const column of columns) {
-                        if (column === 'id') continue;
-                        if (req.body[column]) {
-                            changes[column] = req.body[column];
-                        } else {
-                            changes[column] = null;
-                        }
-                    }
-                    this.db('features').where({ id: req.params.id }).update(changes).then(() => {
-                        this.client.listFeaturesEdited(changes, features[0]);
-                        res.render('lists/features/edit', {
-                            title: `Edit Feature '${changes.name}'`,
-                            subtitle: 'Feature has been updated',
-                            data: changes
-                        });
+                        errors: handle
                     });
+                    res.redirect('/lists/features/' + features[0].id);
```

Comment:
Redirect back to the manage page - that's where you were before you did the edit.